File: app/src/main/java/nic/goi/aarogyasetu/models/LocationObject.java
/*
 * Copyright 2020 Government of India
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *
 */

package nic.goi.aarogyasetu.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LocationObject {

    @SerializedName("lat")
    @Expose
    private String lat;
    @SerializedName("lon")
    @Expose
    private String lon;

    LocationObject(String latitude, String longitude) {
        this.lat = latitude;
        this.lon = longitude;
    }

}

File: app/src/main/java/nic/goi/aarogyasetu/models/LocationObject.kt
package nic.goi.aarogyasetu.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LocationObject internal constructor(
    @field:SerializedName("lat")
    @field:Expose
    private val lat: String, @field:SerializedName("lon")
    @field:Expose
    private val lon: String
)
